include/glow/Base/Type.h

```diff
@@ -181,6 +181,7 @@ inline bool operator==(const ShapeNCHW &LHS, const ShapeNCHW &RHS) {
 enum class ElemKind : unsigned char {
   FloatTy,  // 32-bit float type (float)
   Int8QTy,  // 8-bit quantized type (int8_t)
+  Int16QTy, // 16-bit quantized type (int16_t)
```

Comment:
can you explain the usage of this type? Is that for some intermediate results on int8 tensors?

Reply:
Possible usages:
* intermediate results
* deal with our internal backend constraints (can't give details)
* in the long term we want to extend quantization to support 16 bits